LICENSE

```diff
@@ -1,6 +1,7 @@
 MIT License
 
 Copyright (c) 2023 Georgi Gerganov
+Copyright (c) 2023 SJTU-IPADS
```

Comment:
Sure. But still MIT license. That means I can merge this to mainline llama.cpp without any issues. Read more about licenses here: https://choosealicense.com/